pytest_tests/lib/helpers/node_management.py

```diff
@@ -68,14 +68,19 @@ def stop_storage_nodes(nodes: list[StorageNode]) -> None:
 
 
 @allure.step("Restart storage nodes")
-def restart_storage_nodes(nodes: list[StorageNode]) -> None:
+def restart_storage_nodes(neofs_env: NeoFSEnv) -> None:
     """
-    The function restarts specified storage nodes.
+    The function restarts all storage nodes of the specified neofs env.
     Args:
-       nodes: the list of nodes to restart
+       neofs_env: NeoFSEnv instance
     """
-    stop_storage_nodes(nodes)
-    start_storage_nodes(nodes)
+    for node in neofs_env.storage_nodes:
```

Comment:
can you please add some context to the commit? it is not clear to me why we need it and, moreover, why we cannot use `stop_storage_nodes(neofs_env.storage_nodes)`

Reply:
Fixed. The only needed change here was to update `start_storage_nodes` function.

Reply:
It's not obvious from the commit message _why_ we're changing this. And it's just a workaround, not something critical to the test logic, so this better be explained for people reading the code in a year.